### ISSUE TITLE
May I ask what is the scope of the reward, and is there any plan to update imagereward 2.0

### ISSUE THREAD

**Issue by Enderfga** [THUDM/ImageReward#5]

**Comment by xujz18** [comment]
The reward is normalized to have a mean of 0 and a standard deviation of 1. When testing stable diffusion v1.4 on our metric set, the scope of the reward is observed to have 62.4% of [-1,1] and 98.2% of [-2,2].
We do have a plan to update ImageReward 2.0 based on annotated data of larger scale and higher quality, and considering more abundant dimensions.

**Comment by Enderfga** [comment]
> 

Then I also want to ask which code is used to calculate the blip score in the paper, is there an open source warehouse?

**Comment by xujz18** [comment]
It can be found in https://github.com/THUDM/ImageReward/blob/main/ImageReward/models/BLIPScore.py, which is based on  https://github.com/salesforce/BLIP

**Comment by hkunzhe** [comment]
> The reward is normalized to have a mean of 0 and a standard deviation of 1. When testing stable diffusion v1.4 on our metric set, the scope of the reward is observed to have 62.4% of [-1,1] and 98.2% of [-2,2]. We do have a plan to update ImageReward 2.0 based on annotated data of larger scale and higher quality, and considering more abundant dimensions.

Since the reward is normalized to have a mean of 0 and a standard deviation of 1, why are mean and std 0.16717362830052426 and 1.0333394966054072 respectively?

**Comment by xujz18** [comment]
"mean and std 0.16717362830052426 and 1.0333394966054072 " is to make the reward to have a mean of 0 and a standard deviation of 1.

**Comment by hkunzhe** [comment]
Apologies for the misunderstanding. Allow me to clarify the previous question: Are the mean and std in the code calculated based on the reward from the model's training set?

**Comment by hkunzhe** [comment]
> "mean and std 0.16717362830052426 and 1.0333394966054072 " is to make the reward to have a mean of 0 and a standard deviation of 1.

Apologies for the misunderstanding. Allow me to clarify the previous question: Are the mean and std in the code calculated based on the reward from the model's training set?

**Comment by xujz18** [comment]
In fact, they are calculated based on the reward from the test set.
